third_party/amd/lib/TritonAMDGPUTransforms/AccelerateAMDMatmul.cpp

```diff
@@ -509,15 +508,21 @@ class BlockedToWMMA : public mlir::RewritePattern {
     Type wmmaAccType;
     auto oldRetElemType = oldRetType.getElementType();
     auto aElemType = oldAType.getElementType();
+    auto bElemType = oldBType.getElementType();
     if (oldRetElemType.isIntOrIndex())
       wmmaAccType = rewriter.getIntegerType(32);
     else if (oldRetElemType.isa<mlir::Float16Type, mlir::BFloat16Type>() &&
              aElemType == oldRetElemType)
       wmmaAccType = oldRetElemType;
-    else
+    else if (oldRetElemType.isa<mlir::FloatType>() &&
+             aElemType.getIntOrFloatBitWidth() < 16) {
```

Comment:
For better readability better not to use numbers in the code. Proposed to replace it with named constant. Or some function call that returns bit width of float16.

Reply:
Do you suggest  something like:
``aElemType.getIntOrFloatBitWidth() < rewriter.getF16Type().getWidth()``

I think 16 is more readable. WDYT?